### PR TITLE
[KOGITO-3010] - Add kie-github-ci-bot to kogito-images repository

### PR DESCRIPTION
**Pull request by Kaitou786**
https://issues.redhat.com/browse/KOGITO-3010

Signed-off-by: Tarun Khandelwal <tarkhand@redhat.com>

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a testcase that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Review by spolti — APPROVED** [review]

**Comment by Kaitou786** [comment]
Jenkins test this

**Comment by Kaitou786** [comment]
Jenkins test this

**Comment by Kaitou786** [comment]
Jenkins test this

**Comment by Kaitou786** [comment]
@spolti can be merged?

File: .github/bot-files/comments.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+prFirstTimeContributor: Welcome to the Kogito Community!. Please make sure you've read the [contributors' guide](README.md#contributing-to-kogito-images-repository)
+
+prCiTrigger: Jenkins test this
```
Comment (spolti) [inline .github/bot-files/comments.yml]: can we use /jenkins test like operator?

**Review by spolti — CHANGES_REQUESTED** [review]

File: .github/bot-files/comments.yml (hunk 1 above)
Comment (Kaitou786) [reply .github/bot-files/comments.yml]: Updated in the file, will update it in the pipeline as well

Comment (Kaitou786) [reply .github/bot-files/comments.yml]: Updated in the pipeline settings.

**Comment by Kaitou786** [comment]
@radtriste @sutaakar here also, we are following the same method as  operator so for triggering the pipeline for retest (in case of failure) use `/jenkins test`

**Comment by spolti** [comment]
cool thanks.

**Review by spolti — APPROVED** [review]

**Pull request merged**